frontend/javascripts/dashboard/dataset/dataset_settings_viewconfig_tab.tsx

```diff
@@ -50,23 +115,45 @@ export default function DatasetSettingsViewConfigTab() {
       dataIndex: "comment",
     },
   ];
-  const comments: Partial<Record<keyof DatasetLayerConfiguration, string>> = {
-    alpha: "20 for segmentation layer",
-    min: "Only for color layers",
-    max: "Only for color layers",
-    intensityRange: "Only for color layers",
+  const comments: Partial<Record<keyof DatasetLayerConfiguration, [string, string | null]>> = {
+    alpha: ["20 for segmentation layer", null],
```

Comment:
Why are using arrays to hold the descriptions? Would it not be easier to read to use an object with names keys, e.g. `tooltip`, `shortDescription`?

Reply:
Sure, also fine. Might make thinks also easier to understand :tada: